src/routes/_utils/blurhash.js

```diff
@@ -0,0 +1,14 @@
+import { decode as decodeBlurHash } from 'blurhash'
+
+let canvas
+
+export function decode (blurhash) {
+  const pixels = decodeBlurHash(blurhash, 320, 320)
+
+  if (pixels) {
+    canvas = canvas || document.createElement('canvas')
```

Comment:
Mastodon renders the canvases directly but I though that it's too much overhead to create and destroy that media canvases so I went with this approach.

Reply:
Profile sample with 6x CPU throttling.
![image](https://user-images.githubusercontent.com/2109702/62793650-55639980-bad2-11e9-8e93-bf9ccc4ea4ca.png)

Reply:
I wonder if `toBlob()` with a Blob URL may actually be more performant here than a data URL. I'll play around with it. :)

Reply:
Hm, so the problem with `toBlob()` is it's asynchronous, meaning that there might be a flash of unhidden sensistive media unless we're careful about doing the blurhashing in advance. I might keep noodling on this after merging the PR.